WallsAndHoles/tilemap.h

```diff
@@ -11,22 +11,16 @@
 #include <QSharedPointer>
 #include <QMutex>
 
-struct Properties
```

Comment:
I still don't see much use for having a properties struct (at least at this point).